src/charts/bar.js

```diff
@@ -173,9 +197,9 @@ define(function(require) {
          * Also applies the Margin convention
          * @private
          */
-        function buildContainerGroups(){
+        function buildContainerGroups() {
             let container = svg
-              .append('g')
+                .append('g')
```

Comment:
That 2 spaces indentation has some logic behind. It is a d3 pattern useful to mark when a selection changes.